app/components/applications-app-services/locals.tf

```diff
@@ -62,6 +62,7 @@ locals {
         BACK_OFFICE_INTEGRATION_GET_APPLICATION_CASE_REFERENCES       = var.back_office_integration_get_application_case_references
         BACK_OFFICE_INTEGRATION_GET_DOCUMENTS_CASE_REFERENCES         = var.back_office_integration_get_documents_case_references
         BACK_OFFICE_INTEGRATION_GET_TIMETABLE_CASE_REFERENCES         = var.back_office_integration_get_timetable_case_references
+        BACK_OFFICE_INTEGRATION_GET_S51_CASE_REFERENCES               = var.back_office_integration_get_s51_case_references
```

Comment:
In the ticket I had named this as GET_ADVICE...  rather than S51, it matches the endpoint name it is activating, also S51 is a bit cryptic and PINs acronym/tech speak.